src/app/app.component.ts

```diff
@@ -151,24 +152,27 @@ export class AppComponent implements OnInit, AfterViewInit, CanvasTableSelectLis
     changeDetectorRef: ChangeDetectorRef,
     public mobileQuery: MobileQueryService,
     private swPush: SwPush,
-    private _hotkeysService: HotkeysService) {
-
-      this._hotkeysService.add(new Hotkey(['j', 'k'],
-          (event: KeyboardEvent, combo: string): ExtendedKeyboardEvent => {
-              if (combo === 'k') {
-                  this.canvastable.scrollUp();
-                  combo = null;
-              }
-              if (combo === 'j') {
-                  this.canvastable.scrollDown();
-              }
-              const e: ExtendedKeyboardEvent = event;
-              e.returnValue = false;
-              return e;
-          }));
+    private _hotkeysService: HotkeysService
+  ) {
```

Comment:
Is this change of indentation on purpose?

Reply:
It is – I switched it to 2-spaces to be like the rest of the file, since it was off before that.

Reply:
Its still 4 spaces...

Reply:
Well, it's 4 spaces from column 0, but 2 from the outer scope... I think? :)

The empty line is silly indeed, I can remove it.

Reply:
Commited more explicitely in https://github.com/runbox/runbox7/pull/612/commits/ad923afeb1f09495749d1ad37b164044c3b04900